cmd/minishift/state/instance_dirs.go

```diff
@@ -53,6 +54,7 @@ func GetMinishiftDirsStructure(baseDir string) *MinishiftDirs {
 		Logs:         filepath.Join(baseDir, "logs"),
 		Tmp:          filepath.Join(baseDir, "tmp"),
 		Config:       filepath.Join(baseDir, "config"),
+		Profiles:     filepath.Join(baseDir, "profiles"),
```

Comment:
Did we not include the profiles directory here on purpose? @minishift/minishift-dev

Reply:
@anjannath Ah it should have the profiles directory, Checking the code.

Reply:
@LalatenduMohanty I need this directory to be present even if no profiles have yet been created, as i put a inotify watcher on the directory to monitor it for creation and deletion of profiles.

Reply:
@anjannath We can not add `profiles` to the structure as no profiles except the default profile need to create the directory.  Also if we accept this change every profile will have an extra `profiles` directory e.g. `~/.minishift/profiles/<profile name>/profiles` which will be never used. I would suggest to add code in root.go to create ~/.minishift/profiles directory.

Reply:
@anjannath We can not add profiles to the structure as no profiles except the default profile need to create the directory. Also if we accept this change every profile will have an extra profiles directory e.g. ~/.minishift/profiles/<profile name>/profiles which will be never used. I would suggest to add code in root.go to create ~/.minishift/profiles directory.